sdk/tests/deqp/modules/shared/glsFboCompletenessTests.js

```diff
@@ -213,120 +211,120 @@ goog.scope(function() {
             gl.SRGB8,
             glsFboUtil.formatkey(gl.SRGB, gl.UNSIGNED_BYTE)
         ];
-        
+
         /**
          * s_oesRgb8Rgba8TexFormats
          * @type {Array<number>}
          */
         glsFboCompletenessTests.s_oesRgb8Rgba8TexFormats = [
-            glsFboUtil.formatkey(gl.RGB,  gl.UNSIGNED_BYTE),
+            glsFboUtil.formatkey(gl.RGB, gl.UNSIGNED_BYTE),
             glsFboUtil.formatkey(gl.RGBA, gl.UNSIGNED_BYTE)
         ];
-        
+
         var fmt = glsFboUtil.FormatFlags;
-        
+
         /**
          * s_esExtFormats
          * @type {Array<glsFboUtil.FormatExtEntry>}
          */
         glsFboCompletenessTests.s_esExtFormats = [
             new glsFboUtil.FormatExtEntry(
-                'GL_OES_depth_texture',
+                'gl.OES_depth_texture',
```

Comment:
These extension renames look incorrect. If this string is intended to refer to the extension name then it should be 'OES_depth_texture' rather than 'gl.OES_depth_texture'.